build.assets/makefiles/master/k8s-master/kube-controller-manager.service

```diff
@@ -14,12 +14,15 @@ ExecStart=/usr/bin/kube-controller-manager \
         --master=https://${KUBE_APISERVER}:6443 \
         --logtostderr=true \
         --kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
+        --authentication-kubeconfig=/etc/kubernetes/scheduler.kubeconfig \
```

Comment:
Should controller-manager really be authenticating as scheduler?

Reply:
And for the kubeconfig I will change it here for sure.